paper.md

```diff
@@ -0,0 +1,113 @@
+---
+title: 'Trash AI: A Web GUI for Serverless Computer Vision Analysis of Images of Trash'
+tags:
+  - tensorflow.js
+  - IndexDB
+  - Plastic Pollution
+  - Trash
+  - Litter
+  - AI
+  - Image Classification
+  - Serverless
+  - vue
+  - vuetify
+  - vite
+  - pinia
+authors:
+  - name: Win Cowger
+    orcid: 0000-0001-9226-3104
+    affiliation: 1 # (Multiple affiliations must be quoted)
+  - name: Steven Hollingsworth
+    corresponding: true # (This is how to denote the corresponding author)
+    affiliation: 2
+  - name: Day Fey
+    affiliation: 2
+  - name: Mary C Norris
+    affiliation: 2
+  - name: Walter Yu
+    affiliation: 3
+  - name: Kristiina Kerge
+    affiliation: 4
+  - name: Kris Haamer
+    affiliation: 4
+  - name: Gina Durante
+    affiliation: 2
+  - name: Brianda Hernandez
+    affiliation: 2
+affiliations:
+ - name: Moore Institute for Plastic Pollution Research, USA
+   index: 1
+ - name: Code for Sacramento and Open Fresno, USA
+   index: 2
+ - name: California Department of Transportation, USA
+   index: 3
+ - name: Let's Do It Foundation, Estonia
+   index: 4
+date: 1 September 2022
+bibliography: paper.bib
+
+---
+
+# Summary
+Although computer vision classification routines have been created for trash, they have not been accessible to most researchers due to the challenges in deploying the models. Trash AI is a web GUI (Graphical User Interface) for serverless computer vision classification of batch images with trash in them hosted at www.trashai.org. With a single batch upload and download, a user can automatically describe the types and quantities of trash in all of their images. 
```

Comment:
Second sentence--suggest changing to read as follows:  "...classification of **individual items of trash within images**, hosted at www.trashai.org." (Bolded for emphasis.)  Since the next sentence begins by stating that images can be uploaded as a batch, it's not necessary to state that in the previous sentence, and the original wording was a bit awkward.

Reply:
https://github.com/code4sac/trash-ai/commit/3a33857c8ff03c2b308c58b3b5a7bda3a3a79ba9